documentation/src/main/asciidoc/user_guide/integrations_spring.adoc

```diff
@@ -54,7 +65,8 @@ Now, you will need to add {brandname} and Spring integration module to your clas
             <artifactId>infinispan-spring4-embedded</artifactId>
             <version>${version.spring}</version>
         </dependency>
-        <!-- depending on a use case, one should use Spring Context or Spring Boot jars -->
+        <!-- Use the Spring Boot Starter
```

Comment:
Sorry I did not realise yesterday that these pom is also wrong.  it's version-infinispan and the infinispan-embedded dependency you can remove it